charts/merlin/values.yaml

```diff
@@ -382,11 +390,11 @@ mlflow:
   #   orchestrator: unknown
 
   # Add the following configs when you wish to point to an actual S3 for mlflow to store its artifacts
-  extraEnvs:
+  extraEnvs: {}
     #   AWS_ACCESS_KEY_ID: YOURACCESSKEY
     #   AWS_SECRET_ACCESS_KEY: YOURSECRETKEY
     #   AWS_DEFAULT_REGION: ap-southeast-2
-    MLFLOW_S3_ENDPOINT_URL: http://minio.minio.svc.cluster.local:9000
+    #   MLFLOW_S3_ENDPOINT_URL: http://minio.minio.svc.cluster.local:9000
```

Comment:
Should we comment here not to include this env var, use `Values.minio.enabled` instead?